### DISCUSSION TITLE
Preact build with only hooks?

### DISCUSSION THREAD

**Discussion by kethan** [preactjs/preact#3326] [General]
Is there a way where I just want hooks and functional components without those class components and mount functions? I just want a minimal build. Is it possible?

something like https://github.com/yisar/fre

**Comment by developit** [comment]
@kethan - not yet, but we're working to support this exact usage in Preact 11. Some of the architectural changes have already been implemented (eg: https://github.com/preactjs/preact/pull/3103)

**Comment by kethan** [comment]
> @kethan - not yet, but we're working to support this exact usage in Preact 11. Some of the architectural changes have already been implemented (eg: #3103)

Wow feeling excited already. When is preact 11 releasing? :( or Can I use that beta version?

**Comment by developit** [comment]
We don't have a release date set yet, and aren't yet publishing alphas.
